guides/v3.7.0/tutorial/autocomplete-component.md

```diff
@@ -413,15 +413,10 @@ module('Integration | Component | list-filter', function(hooks) {
 });
 ```
 
-Finally we add a `settled` call at the end of our test to assert the results.
+Finally we add a `settled` call at the end of our test to assert the results. Ember's [settled helper](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#settled)
+waits for all asynchronous tasks to complete.
 
-Ember's [settled helper](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#settled)
-waits for all asynchronous tasks to complete before running the given function callback.
-It returns a promise that we also return from the test.
-
-If you return a promise from a QUnit test, the test will wait to finish until that promise is resolved.
-In this case our test completes when the `settled` helper decides that processing is finished,
-and the function we provide that asserts the resulting state is completed.
+In this case our assertions will not be made until the `settled` helper decides that processing is finished.
```

Comment:
I think I prefer the old phrasing of this paragraph 🤔

Reply:
I agree but the old content seemed to no longer apply directly to the example because we are no longer returning a promise from the QUnit tests. I wasn't sure quite how to reword it though.

Reply:
that's not entirely true. we *are* still returning a Promise, it's just not directly visible because we're using an async function.

Reply:
@Turbo87 I reverted the content back to its original wording.